std_msgs/Header header
uint16 status
uint16 sample_counter
float32 x
float32 y
float32 z
float32 pitch
float32 roll
float32 yaw
float32 alpha
float32 beta
float32 q3
float32 q2
float32 q1
float32 q0

